src/components/calculator.js

```diff
@@ -1,78 +1,153 @@
-import React from 'react';
+/* eslint-disable react/jsx-closing-bracket-location */
```

Comment:
- [ ] `required`
- [ ] Amazing work so far 👏 
- [ ] please enable the disabled linters because it defeats the purpose of using linters 🥂